exchanges/hitbtc/hitbtc_test.go

```diff
@@ -467,7 +467,7 @@ func TestGetDepositAddress(t *testing.T) {
 		}
 	}
 }
-func setupWsAuth(t *testing.T) {
+func setupWsAuth(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting